ros_ws/src/navigation_stack/car_cartographer/launch/cartographer_offline_fast.launch

```diff
@@ -0,0 +1,41 @@
+<!--
+  Copyright 2016 The Cartographer Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!-- Modified by the F1/10 Autonomous Racing Project Group -->
+
+<launch>
+  <param name="/use_sim_time" value="true" />
+
+  <arg name="ros_version" default="kinetic"/>
+
+  <group if="$(eval ros_version=='kinetic')">
+    <node name="cartographer_offline_node" pkg="cartographer_ros" type="cartographer_offline_node" args="
+          -configuration_directory $(find car_cartographer)/config
+          -configuration_basename cartographer_config_kinetic.lua
+          -urdf_filename $(find car_cartographer)/files/racer.urdf
+          -bag_filenames $(arg bag_filenames)" output="screen" required="true">
+    </node>
+  </group>
+
+  <group if="$(eval ros_version=='melodic')">
+    <node name="cartographer_offline_node" pkg="cartographer_ros" type="cartographer_offline_node" args="
+          -configuration_directory $(find car_cartographer)/config
+          -configuration_basenames cartographer_config_melodic.lua
+          -urdf_filenames $(find car_cartographer)/files/racer.urdf
+          -bag_filenames $(arg bag_filenames)" output="screen" required="true">
+    </node>
+  </group>
```

Comment:
Can this be simplified by doing something like
``` -configuration_basenames cartographer_config_$(arg ros_version).lua ```? If this is viable, also check the other launch files.

Reply:
Done. Note that this was not possible in `cartographer_offline_fast.launch` because there are more changes than that between the two versions ("urdf_filename" vs. "urdf_filenames").